universal-login-relayer/lib/services/WalletService.ts

```diff
@@ -1,28 +1,43 @@
-import ProxyContract from 'universal-login-contracts/build/Proxy';
-import WalletContract from 'universal-login-contracts/build/Wallet';
-import WalletMasterContract from 'universal-login-contracts/build/WalletMaster';
-import LegacyWallet from 'universal-login-contracts/build/LegacyWallet';
+const WalletContract = require('universal-login-contracts/build/Wallet');
+const WalletMasterContract = require('universal-login-contracts/build/WalletMaster');
+const ProxyContract = require('universal-login-contracts/build/Proxy');
+const LegacyWallet = require('universal-login-contracts/build/LegacyWallet');
 import {hasEnoughToken, isAddKeyCall, getKeyFromData, isAddKeysCall} from '../utils/utils';
-import {utils, Contract, ContractFactory} from 'ethers';
+import {utils, ContractFactory, Wallet, providers} from 'ethers';
 import defaultDeployOptions from '../config/defaultDeployOptions';
+import ENSService from './ensService';
+import AuthorisationService from './authorisationService';
+import {EventEmitter} from 'fbemitter';
 
+declare interface Message {
```

Comment:
Already declared in:
https://github.com/UniversalLogin/UniversalLoginSDK/blob/master/universal-login-sdk/lib/sdk.ts#L11

Let's move it to common same way in this PR:
https://github.com/UniversalLogin/UniversalLoginSDK/pull/387/files#diff-3b1d3523abf5d5b79d2830b0088f9715

Note: Refactor following as well:
https://github.com/UniversalLogin/UniversalLoginSDK/blob/master/universal-login-contracts/lib/calculateMessageSignature.ts#L3